src/relo.coffee

```diff
@@ -16,13 +16,17 @@ spawn = ->
   program = options.command[0]
   argv    = options.command[1..]
 
+  if ! argv.length && program.search(' ') != -1
+    [program, argv...] = program.split(' ')
+
   subproc = cp.spawn program, argv,
     cwd     : process.cwd()
-    stdio   : 'inherit'
     detached: true
 
   debug "Started #{program}"
 
+  subproc.stdout.on 'data', (buffer) -> console.log(buffer.toString().trim())
```

Comment:
Hi!
It's kind of a quick fix, stringifying the buffer on OSX gave me the outputted line with a newline at the end.

In retrospect, I think it might be wiser to use `process.stdout.write(buffer.toString())` instead